infrastructure/main.tf

```diff
@@ -47,7 +47,7 @@ locals {
 }
 
 module "citizen-frontend" {
-  source = "git@github.com:contino/moj-module-webapp.git"
+  source = "git@github.com:hmcts/moj-module-webapp.git?ref=RPE-389/local-cache"
```

Comment:
My only concern is that we using something that has not been merged into stable codebase and will be used in production. On one hand it worries it that it might not been properly tested. On other hand that somebody might change something on feature branch without realising that services are using it in prod.